e2e/npm_translate_lock/MODULE.bazel

```diff
@@ -1,20 +1,18 @@
-module(
-    name = "e2e_npm_translate_lock",
-    version = "0.0.0",
-    compatibility_level = 1,
-)
-
-bazel_dep(name = "bazel_skylib", version = "1.5.0")
-bazel_dep(name = "aspect_bazel_lib", version = "2.7.1")
-bazel_dep(name = "aspect_rules_js", version = "0.0.0")
-bazel_dep(name = "platforms", version = "0.0.8")
-
+bazel_dep(name = "aspect_rules_js", version = "0.0.0", dev_dependency = True)
 local_path_override(
     module_name = "aspect_rules_js",
     path = "../..",
 )
 
-npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm")
+bazel_dep(name = "aspect_bazel_lib", version = "2.7.1", dev_dependency = True)
+bazel_dep(name = "bazel_skylib", version = "1.5.0", dev_dependency = True)
+bazel_dep(name = "platforms", version = "0.0.9", dev_dependency = True)
+
+npm = use_extension(
```

Comment:
Why were some of these added?

Reply:
These are moved from above, re-ordered, and `dev_dependency = True` is added to them

Reply:
Oh I see, adding `dev_dependency` made it 3 lines so I didn't see the single line removed while quickly scanning it 😅